tests/test_diagnosis_routes.py

```
@@ -0,0 +1,142 @@
+    device_uid = "uid_any"
+
+    transport = ASGITransport(app=app)
+    async with AsyncClient(transport=transport, base_url="http://test") as ac:
+        resp = await ac.get("/ai/diagnosis/drowsiness", params={"deviceUid": device_uid})
+
+    assert resp.status_code == 500
+    data = resp.json()
+    assert data["error"]["message"] == "model_not_loaded"
+
+@pytest.mark.asyncio
+async def test_queue_not_found():
```

Ruff (0.11.9)

Missing return type annotation for public function `test_queue_not_found`

Add return type annotation: `None`

(ANN201)